xbmc/settings/AdvancedSettings.cpp

```diff
@@ -280,6 +280,7 @@ void CAdvancedSettings::Initialize()
   m_strMusicLibraryAlbumFormat = "";
   m_prioritiseAPEv2tags = false;
   m_musicItemSeparator = " / ";
+  m_musicArtistSeparators = { ";", ":", "|", "#", " feat. ", " ft. " };
```

Comment:
Thanks for taking an interest @garbear, there are some crazy artist names out there!

This is one reason why the music artist separators are configurable. If you have "10 Ft. Ganja Plant" in your music collection then you can drop " ft " as a music artist separator, or use ARTISTS tag or tag with Musicbrainz IDs (having added it to their database) in those songs. 

Many users have "artist1 ft. artist2" in tags, so I think it reasonable to have " ft. " as a default setting.